ffi/src/lib.rs

```diff
@@ -1054,6 +1062,9 @@ pub unsafe extern "C" fn command(
     for command_arg in arg_vec {
         cmd.arg(command_arg);
     }
+    if span_ptr != 0 {
```

Comment:
If yes, you can do so
```suggestion
    if !span_ptr.is_null() {
```